ggshield/cmd/config/config_list.py

```diff
@@ -1,35 +1,37 @@
+import json
 from typing import Any, Tuple
 
 import click
 
-from ggshield.cmd.utils.common_options import add_common_options
+from ggshield.cmd.utils.common_options import add_common_options, json_option
 from ggshield.cmd.utils.context_obj import ContextObj
 
 from .constants import DATETIME_FORMAT, FIELDS
 
 
 @click.command()
 @click.pass_context
+@json_option
```

Comment:
Can you also add `@text_json_format_option`? It adds the `--format` option which is now provided by all commands with a `--json` option.